### PR TITLE
Tests fixed

### PR DESCRIPTION
**Pull request by clausmichele**
I fixed the tests. Does it make sense to refer to the official openEO processes list? Is it fine to specify the address in the tests script or should we have it in a config file?

### CODE REVIEW

File: src/openeo_pg_parser/utils.py [hunk 1]
```diff
@@ -75,9 +75,13 @@ def load_processes(src):
             filepaths = glob.glob(os.path.join(src, "*.json"))
             process_list = [load_json_file(filepath) for filepath in filepaths]
         elif isinstance(src, str) and url_is_valid(src):
+            # Is it the URL of a JSON file or a /processes endpoint?
             r = requests.get(url=src)
             data = r.json()
-            process_list = data['processes']
+            if src.split('.')[-1].lower() == 'json':
```
Comment (m-mohr) [inline src/openeo_pg_parser/utils.py:81]: I think this assumption may not always be true. I guess it's better to simply check whether you get an object with a "processes" property or whether you get an array and then handle it accordingly.

File: tests/test_graph.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import unittest
 from openeo_pg_parser.translate import translate_process_graph
 
+OPENEO_PROCESSES_ENDPOINT = "https://processes.openeo.org/processes.json"
```
Comment (m-mohr) [inline tests/test_graph.py:5]: To ensure that tests are not failing due to a new release, I'd propose to link against a specific version.

```suggestion
OPENEO_PROCESSES_ENDPOINT = "https://processes.openeo.org/1.2.0/processes.json"
```

File: tests/test_translate.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 import unittest
 from openeo_pg_parser.translate import translate_process_graph
 
+OPENEO_PROCESSES_ENDPOINT = "https://processes.openeo.org/processes.json"
```
Comment (m-mohr) [inline tests/test_translate.py:5]: same here 
```suggestion
OPENEO_PROCESSES_ENDPOINT = "https://processes.openeo.org/1.2.0/processes.json"
```

**Comment by clausmichele** [comment]
@m-mohr I've applied all your proposals

**Review by m-mohr — APPROVED** [review]

**Comment by m-mohr** [comment]
Thank you

**Pull request merged**